source/blog/2018-07-06-the-ember-times-issue-54.md

```diff
@@ -49,8 +49,16 @@ get all the TypeScript neatness to your test suite today and type on! ⌨
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Twiddle Your Thumbs No More](#section-url)
```

Comment:
Would it make sense to add add https://ember-twiddle.com/ as the section url as well?

Reply:
Yup! Missed that thanks!